### ISSUE TITLE
`Object.observe()` is going away 

### ISSUE THREAD

**Issue by cloverich** [chrisabrams/dragon#8]
Hey -- had been considering doing something similar with Backbone and relying on Object.observe, but ran across this discussion and apparently [Object.observe is going away](https://esdiscuss.org/topic/an-update-on-object-observe) :(

**Comment by chrisabrams** [comment]
Yes I'm not happy about that because proxies don't compile down to ES5.

**Comment by kjantzer** [comment]
@cloverich You beat me to it! Was just about to post this issue.

**Comment by chrisabrams** [comment]
Most likely going to use event emitter.

**Comment by enjikaka** [comment]
@cloverich @kjantzer Did you also get the thought of making an issue here after reading the latest issue of "JavaScript Weekly"? :dancer:

**Comment by cloverich** [comment]
@enjikaka yes indeed :D @chrisabrams the full thread includes some interesting discussion on other lib authors take on the subject (including why some shied away from Object.observe)

**Comment by chrisabrams** [comment]
It was fun to play with, but I definitely had some issues with it. I don't do a lot of work/projects with data-binding, so it hasn't been something I've focused on much.

**Comment by chrisabrams** [comment]
PS I have no idea how my project ended up in JS weekly.

**Comment by enjikaka** [comment]
Here's the link if you haven't read it already @chrisabrams 

http://javascriptweekly.com/issues/257

**Comment by 91bananas** [comment]
This is non-constructive for this discussion, but the jsweekly thing is just a list of articles from around the web that the maintainer hand selects. I mostly see things on hacker news first throughout the week, then see them on jsweekly every Friday.

As for this project kudos to you, I've often thought backbone is so close to being super-awesome if only it had come a little later in the js timeline, and could have taken some of this stuff into account. You have a lot of very cool ideas in your readme. Keep it up!

**Comment by joezimjs** [comment]
I think the best alternative to Object.observe is to use getters and setters like Ampersand's models.
https://github.com/ampersandjs/ampersand-model

**Comment by joezimjs** [comment]
Speaking of which, it might be a good idea to work with Ampersand to convert to ES6 classes and such things like that because they already have a good start on some of these changes, plus they've made each component a separate Node module so you only need to import the pieces you need instead of the entire library.

**Comment by wyqydsyq** [comment]
https://medium.com/@mweststrate/object-observe-is-dead-long-live-mobservable-observe-ad96930140c5

This 'Mobservable' library could be useful.

**Comment by chrisabrams** [comment]
@wyqydsyq seems pretty cool, will take a look.

@joezimjs ampersand models also look promising.

Much to figure out :O

**Comment by sylvainpolletvillard** [comment]
ES5 getters and setters are supported for a long time, even in IE9. I use them as a basic substitute for Object.observe and Array.observe:
https://gist.github.com/sylvainpolletvillard/2310b297adb063c53be7

This is the most obvious choice to replace Backbone get/set
